code/datums/supply_packs/food.dm

```diff
@@ -55,11 +55,140 @@
 					)
-	cost = 10
+	cost = 7
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "\improper Tomato crate"
 	group = "Food"
 
+/datum/supply_packs/soybeans
+	name = "soybeans (x20)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper soybeans crate"
+	group = "Food"
+
+/datum/supply_packs/soybeans
+	name = "cocoapod,sugarcane (x10)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/cocoapod,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					/obj/item/reagent_container/food/snacks/grown/sugarcane,
+					)
+	cost = 7
```

Comment:
this value is inherited from the parent /supply_packs and does not need to be written down again. You should be using defines rather than numbers anyway.

Reply:
didn't understood what you meant with this one sorry... if it's the cost = 7 it's remove now since i made a parent object called plants